src/views/article.hbs

```diff
@@ -0,0 +1,5 @@
+<section class="container">
+<div class="article_img"><img src="https://static1.squarespace.com/static/57e6d568d482e9a8f589ebc7/t/57e7e1a3e4fcb5e38d0388f4/1474814526471/?format=300w" alt="hospital"></div>
+<div class="article_title" >Lorem ipsum dolor sit amet, consectetuer adipiscing</div>
```

Comment:
maybe you should use one of the heading tags for the paragraph title like h1,h2,...